src/modules/search-result/search-result.jsx

```diff
@@ -0,0 +1,132 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import Metadata from './metadata'
+import AuthorList from './author'
+import styles from './styles.css'
+
+import { Card, Heading, Link } from 'elements'
+import { classNames } from 'utils'
+
+const texts = {
+  pdfAvaliable: `Get PDF`,
+  linkAvailable: 'Access PDF',
+  metadataOnly: 'No full text',
+}
+
+const PREFIX = 'work'
+
+const labels = {
+  author: 'Author',
+  fieldOfStudy: 'Field of study',
+  publicationDate: 'Publication date',
+  publicationVenue: 'Publication venue',
+}
+
+const formatDate = (date) => date?.toString()
+
+const SearchResult = ({
+  children,
+  id,
+  className,
+  data: {
+    title,
+    author = [],
+    publicationDate,
+    publicationVenue,
+    fieldOfStudy,
+    metadataLink,
+    fullTextLink,
+    thumbnailUrl,
+  } = {},
+  ...htmlProps
+}) => {
+  const idFor = (scope) => `${id}-${scope}`
+  return (
+    <Card
+      id={`${PREFIX}-${id}`}
+      className={classNames.use(styles.container).join(className)}
+      itemScope
+      itemType="https://schema.org/ScholarlyArticle"
+      {...htmlProps}
+    >
+      <Heading level="3" className={styles.title} itemProp="name">
+        {metadataLink ? <Link href={metadataLink}>{title}</Link> : title}
+      </Heading>
+
+      <Metadata>
+        <Metadata.Item id={idFor('author')} label={labels.author}>
+          <AuthorList aria-label={labels.author}>
+            {author.map(({ name }) => (
+              <AuthorList.Item>{name}</AuthorList.Item>
```

Comment:
you should pass down id here

Reply:
Why? Which ID?

I made 2 ones: document ID that should be under `data.id` and the `id` attribute which is used in the DOM.

Reply:
The same as here https://github.com/oacore/design/pull/350